### ISSUE TITLE
REVIEW ME: VBET 0.8.2 Runs from Dec6

### ISSUE THREAD

**Issue by MattReimer** [Riverscapes/riverscapes-tools#669]
Just a quick little summary from last night's runs. We can close this after we review them. I'm not keen on letting these tickets build up. 

https://docs.google.com/spreadsheets/d/1ZtuVuxLqjAxFU-MrxOX6ZjL8jtduqcyNwyuZvaPRaPI/edit#gid=1279742273

HUC8 | Version | Duration |   |   | notes
-- | -- | -- | -- | -- | --
0501000301 | 0.8.2 | 1:53:19 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/37120fd2-bb3f-4612-98fd-7dd46b86e880) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/37120fd2-bb3f-4612-98fd-7dd46b86e880) |  
0502000405 | 0.8.2 | 2:18:19 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/ce0c38f0-10ab-4c33-9d09-832f311a4a92) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/ce0c38f0-10ab-4c33-9d09-832f311a4a92) |  
0504000306 | 0.8.2 | 2:20:43 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/fc2e9147-eafd-4d3b-8819-3a977f092e96) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/fc2e9147-eafd-4d3b-8819-3a977f092e96) |  
0514020112 | 0.8.2 | 5:07:54 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/9a4c30f2-02bb-4794-b2f1-2f3644dc2f67) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/9a4c30f2-02bb-4794-b2f1-2f3644dc2f67) |  
0601020203 | 0.8.2 | 2:11:31 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/e40297fb-213c-447e-9075-839f42ff9d08) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/e40297fb-213c-447e-9075-839f42ff9d08) |  
0601020704 | 0.8.2 | 6:26:23 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/883b85ba-fee1-4308-8f75-996e309b6e6f) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/883b85ba-fee1-4308-8f75-996e309b6e6f) |  
0702000504 | 0.8.2 | 1:44:13 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/f75d8c84-666f-4133-af8e-e7be7a338bfb) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/f75d8c84-666f-4133-af8e-e7be7a338bfb) |  
0705000202 | 0.8.2 | 7:06:03 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/df43b87d-f838-4d0f-8736-574836fc8466) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/df43b87d-f838-4d0f-8736-574836fc8466) |  
1003020102 | 0.8.2 | 3:35:05 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/4e88a592-d249-4eed-8bf5-b24d0525b6e2) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/4e88a592-d249-4eed-8bf5-b24d0525b6e2) |  
1005001505 | 0.8.2 | 8:03:44 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/c539378f-8259-4353-b99f-46d6c69b0073) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/c539378f-8259-4353-b99f-46d6c69b0073) | Not quite on the CDN border. Might need a different HUC10 before we can close #639
1019000209 | 0.8.2 | 2:16:51 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/d95446c6-d09b-476f-8b01-687e693bf547) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/d95446c6-d09b-476f-8b01-687e693bf547) |  
1019000309 | 0.8.2 | 4:12:33 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/341b5dd9-ca74-4554-819e-549d49c22f8b) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/341b5dd9-ca74-4554-819e-549d49c22f8b) |  
1019000402 | 0.8.2 | 2:45:32 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/e9e08e63-01b1-4961-94a9-7557e32363f6) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/e9e08e63-01b1-4961-94a9-7557e32363f6) |  
1019000507 | 0.8.2 | 3:32:48 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/16cce8a4-6741-4a1d-8e69-df333474351e) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/16cce8a4-6741-4a1d-8e69-df333474351e) |  
1019001003 | 0.8.2 | 2:19:47 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/1c4b15c6-1909-4b2e-9574-25a10c4c3447) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/1c4b15c6-1909-4b2e-9574-25a10c4c3447) |  
1601020101 | 0.8.2 | 2:42:32 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/9656159a-5e16-4d37-85f8-de7e74da5d40) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/9656159a-5e16-4d37-85f8-de7e74da5d40) | related to #659
1601020201 | 0.8.2 | 0:42:54 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/96e12070-6fc8-42d6-9807-025263d696cf) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/96e12070-6fc8-42d6-9807-025263d696cf) |  
1601020204 | 0.8.2 | 1:37:54 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/4361ef01-3894-4efd-b6cf-698fd1298751) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/4361ef01-3894-4efd-b6cf-698fd1298751) |  
1601020303 | 0.8.2 | 2:11:15 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/236166d1-95d5-4246-9cf8-e45e155dfeb4) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/236166d1-95d5-4246-9cf8-e45e155dfeb4) |  
1706030402 | 0.8.2 | 0:00:00 | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/833985c1-e8cc-4b55-bf46-84ad39e1c792) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/833985c1-e8cc-4b55-bf46-84ad39e1c792) | Potential problem

**Comment by MattReimer** [comment]
Closing this in favour of updating #651
